README.md

```diff
@@ -7,3 +7,8 @@ A Python Client for the GameBench API
 [![Gitter](https://badges.gitter.im/bigfishgames/GameBench-API-PyClient.svg)](https://gitter.im/bigfishgames/GameBench-API-PyClient?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 Please check out our [ZenHub Board](https://app.zenhub.com/workspaces/gamebenchapi-pyclient-5cabf535a736c27636b0283d/board?repos=180245554) for open issues.
+
+For full documentation, go to the `docs`_ page.
+
+.. _docs: https://gamebenchapi-pyclient.readthedocs.io/
+.. _gitHub: https://github.com/bigfishgames/GameBenchAPI-PyClient
```

Comment:
See the ZenHub Board link above for MD link syntax.

Reply:
Fixed in latest push